src/app/kvm/components/LXDVMsTable/VMsTable/VMsTable.tsx

```diff
@@ -271,7 +266,13 @@ const VMsTable = ({
             ),
           },
         ]}
-        rows={generateRows(paginatedVms, getResources, tags, getHostColumn)}
+        rows={generateRows(
```

Comment:
Might make sense to refactor these arguments to a single object at this stage.

Reply:
Done!